packages/tracking/src/trackClick.ts

```diff
@@ -14,6 +14,11 @@
  * limitations under the License.
  */
 
+/**
+ * External dependencies
+ */
+import type { MouseEvent } from 'react';
```

Comment:
Where is this used?

Reply:
30 lines below.

It was mistakenly using the wrong global `MouseEvent` type instead of the React-provided one.